Documentation/api.md

```diff
@@ -277,27 +277,29 @@ The first terminal should get the notification and return with the same response
 However, the watch command can do more than this.
 Using the index, we can watch for commands that have happened in the past.
 This is useful for ensuring you don't miss events between watch commands. 
-Typically, we watch again from the (modifiedIndex + 1) of the node we got.
+Typically, we watch again from the `modifiedIndex + 1` of the node we got.
 
-Let's try to watch for the set command of index 7 again:
```

Comment:
um... actually i still want to say user can get the previous event immediately. @yichengq @barakmich  any opinion?